apps/meteor/server/lib/refreshLoginServices.ts

```diff
@@ -1,11 +1,20 @@
 import { ServiceConfiguration } from 'meteor/service-configuration';
 
+import { notifyOnLoginServiceConfigurationChangedById } from '../../app/lib/server/lib/notifyListener';
 import { loadSamlServiceProviders } from '../../app/meteor-accounts-saml/server/lib/settings';
 import { updateCasServices } from './cas/updateCasService';
 import { updateOAuthServices } from './oauth/updateOAuthServices';
 
 export async function refreshLoginServices(): Promise<void> {
 	await ServiceConfiguration.configurations.removeAsync({});
 
-	await Promise.allSettled([updateOAuthServices(), loadSamlServiceProviders(), updateCasServices()]);
+	const [oAuthServices] = await Promise.allSettled([updateOAuthServices(), loadSamlServiceProviders(), updateCasServices()]);
+
+	if (oAuthServices.status === 'fulfilled') {
+		await Promise.all(
+			oAuthServices.value.map((service) =>
+				notifyOnLoginServiceConfigurationChangedById(service._id, service.deleted ? 'removed' : 'inserted'),
```

Comment:
previously we always sent `updated` instead of `inserted`.. not sure which one is the correct one though